### PR TITLE
Suppress subfield 5 on notes

### PR DESCRIPTION
**Pull request by jamesgorrie**
Fixes https://github.com/wellcomecollection/platform/issues/5024

Subfield 5 on notes field is exclusively used for system information, so should be supressed to avoid confusing leaks like [this](https://wellcomecollection.org/works/thjctx62) <= note the exhibition note has `UkLW` in it

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotes.scala [hunk 1]
```diff
@@ -57,7 +58,8 @@ object SierraNotes extends SierraDataTransformer with SierraQueryOps {
       .map {
         case (vf, NotesField(createNote, suppressedSubfields)) =>
           val contents =
-            vf.subfieldsWithoutTags(suppressedSubfields.toSeq: _*)
+            vf.subfieldsWithoutTags(
+                universallySuppressedSubfields.toSeq ++ suppressedSubfields.toSeq: _*)
```
Comment (alexwlchan) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotes.scala:62]: Do you need to include `suppressedSubfields` at all now? I wonder if it's worth keeping, or if we should remove it?

File: pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotesTest.scala [hunk 2]
```diff
@@ -98,19 +98,25 @@ class SierraNotesTest
...
+    val bibData = createSierraBibDataWith(
+      varFields = varFields.toList
     )
-    SierraNotes(bibData) shouldBe List(BindingInformation("Main bit."))
+
+    val notes = SierraNotes.notesFields.values
+      .map(notesField => notesField.createNote("Main bit."))
+      .toList
+
+    SierraNotes(bibData) should contain theSameElementsAs (notes)
```
Comment (alexwlchan) [inline pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotesTest.scala:119]: nit: No need for parens here:

```suggestion
    SierraNotes(bibData) should contain theSameElementsAs notes
```

**Review by alexwlchan — APPROVED** [review]

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotes.scala (hunk 1 above)
Comment (jamesgorrie) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotes.scala]: Ummed and ahhed about it - I've removed it to be clear about the intention.

**Comment by jamesgorrie** [comment]
Cheers @alexwlchan - my brain was frazzled yesterday.

**Pull request merged**